### ISSUE TITLE
Schema design for common master data

### ISSUE THREAD

**Issue by ppetpadriew** [ppetpadriew/turbo-erp-api#1]
**Common Master Data**
- [x] Item
    - [x] Inventory unit relation

    - [x] Weight unit relation

- [x] Unit

- [x] Warehouse

- [x] Business Partner

- [x] Currency

- [x] Address

Review the following once done
- Unique
- FK (Relation)
- column name

- Create migration files for these table

- [x] Item

- [x] Unit

- [x] Warehouse

- [x] Business Partner

- [x] Currency

- [x] Address

**Comment by ppetpadriew** [comment]
Once the design is done, please push `schema.sql` file to this repo.

**Comment by ppetpadriew** [comment]
[common-master-data.zip](https://github.com/ppetpadriew/turbo-erp-api/files/1563069/common-master-data.zip)

**Comment by ppetpadriew** [comment]
We decided to import `.sql` directly into the container and push the container state to the docker hub instead. So migration files are not needed now.
